backend/src/main/java/eventlistener/service/event/EventService.java

```diff
@@ -44,14 +42,14 @@ public Event addEvent(Event eventToAdd) {
 
 
     public boolean eventsExist(List<Long> listenEvents) {
-        List<Event> listOfEvents = (List<Event>) eventRepo.findAllById(listenEvents);
+        List<Event> listOfEvents = eventRepo.findAllById(listenEvents);
         return listOfEvents.size() >= listenEvents.size();
```

Comment:
don't you wanna check which of the events doesn't exist and possibly give feedback about it?